docs/source/bricks/embedding.rst

```diff
@@ -1,21 +1,21 @@
-########
+#########
 Embedding
-########
+#########
 
 EmbeddingEncoder classes in ``unstructured`` use document elements detected
```

Comment:
Here I would not refer to "EmbeddingEncoder" classes in camel case like that. It makes it sound like a Python class, but we don't define a class with that name. I would either go with something descriptive in plain English like "Embedding encoder classes..." or refer to the actual base class `BaseEmbeddingEncoder` (and in that case use monospace font to indicate this is actually something from the code).